protocol/lib/bank.go

```diff
@@ -0,0 +1,13 @@
+package lib
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func MustParseCoinsNormalized(coinStr string) sdk.Coins {
```

Comment:
nit: add some test cases for this

Reply:
nit: should we add a quick doc comment (i.e. let CoPilot draft one?)